mitosheet/mitosheet/steps_manager.py

```diff
@@ -232,6 +223,17 @@ def __init__(
                 preprocess_step_performers.preprocess_step_type()
             ] = execution_data       
 
+        # We set the original_args_raw_strings. If we later have an args update, then these
+        # are overwritten by the args update (and are actually correct). But since we don't 
+        # always have an args update, this is the best we can do at this point in time. Notably, 
+        # these are required to be set for transpiling arguments
+        self.original_args_raw_strings: List[str] = []
+        for index, arg in enumerate(args):
```

Comment:
Ah -- here's our bug. Args is changed in the above lines by the preprocessing steps, as all the args get converted to dataframes by the preprocessing steps. I think you can just `enumerate(self.original_args)` - and then everything should work!  Or, maybe better would be renaming the `args` overwriting to make it so the preprocessing steps use a variable called `processed_args` or something. Your call.

I think there was one more test above I thought would be good, namely with some None value's that get filtered out, that we should test for (b/c the df_names might be at the wrong index otherwise). 

Otherwise, once we add that test and fix any resulting bugs, LGTM.